Makefile

```diff
@@ -59,8 +59,9 @@ ec:
 libsodium:
 ifeq (,$(wildcard ${STATIC_LIB}.*))
 	@echo Static libsodium was not found at ${STATIC_LIB} so compiling libsodium from source.
-	@cd $(LIBSODIUM_DIR)/ && ./configure  \
-		--enable-static --enable-shared --with-pic --prefix="$(INSTALL_DIR)"
+	@cd $(LIBSODIUM_DIR)/ && ./autogen.sh
```

Comment:
Thanks!
Why add autopen? I was using it in the past and it was causing problems on some platforms. Also libsodium does not require it for manual installation. 
https://download.libsodium.org/libsodium/content/installation/